Frame.cc

```diff
@@ -1457,7 +1457,7 @@ void Frame::SaveFile(const std::string& root_folder, const CARTA::SaveFile& save
         success = true;
     } else if (output_file_type == CARTA::FileType::FITS) {
         // Remove the old image file if it has a same file name
-        casacore::Bool ok = casacore::ImageFITSConverter::ImageToFITS(message, *image, output_filename.string(), 64, casacore::True,
+        casacore::Bool ok = casacore::ImageFITSConverter::ImageToFITS(message, *image, output_filename.string(), 64, casacore::False,
             casacore::True, -32, 1.0, -1.0, casacore::True, casacore::False, casacore::True, casacore::False, casacore::False,
```

Comment:
This now sets preferVelocity=False, opticalVelocity=True.  The comment for this parameter says "If writing a velocity, use the optical definition (otherwise use radio)".  To get the correct FITS headers in FileExtInfoLoader.cc, I checked this with:
    casacore::MDoppler::Types vel_doppler(coord_sys.spectralCoordinate().velocityDoppler());
    if ((vel_doppler == casacore::MDoppler::Z) || (vel_doppler == casacore::MDoppler::OPTICAL)) {
        optical_velocity = true;
    }
But @kswang1029 you said the VRAD headers were correct?

Reply:
uh... I will need to double check again the case of optical velocity definition!

Reply:
Tracked the use of the opticalVelocity parameter down to casacore::FITSSpectralUtil::toFITSHeader:
	    // OPTICAL
	    refVelocity = -C::c * (1.0 - restfreq / refFrequency);
	    velocityIncrement = -C::c * (1.0 - restfreq / (refFrequency + freqIncrement)) - refVelocity;

	    // RADIO
	    refVelocity = -C::c * (refFrequency/restfreq - 1.0);
	    velocityIncrement = -C::c * ((refFrequency + freqIncrement)/restfreq - 1.0) - refVelocity;

Reply:
The header parser in CARTA is doing something unexpectedly when loading an image with VOPT axis (test image at set_save_image/M17_SWex_VOPT.fits which is generated with CASA-exportfits from the image with FREQ). In CARTA we see the axis is somehow converted to VRAD.... Below are screenshots from ds9 and CARTA with the same image "M17_SWex_VOPT.fits"
![Screen Shot 2020-12-24 at 10 01 08 AM](https://user-images.githubusercontent.com/20819712/103050606-ac86e300-45cf-11eb-914b-8e95fb741168.png)
.
![Screen Shot 2020-12-24 at 10 01 37 AM](https://user-images.githubusercontent.com/20819712/103050600-aa248900-45cf-11eb-9c1c-e9164de46960.png)

We will need to fix this first to see the "correct" header first.

From the implementation point of view, if the input cube is VOPT, the subimage export needs to be in VOPT too. In short, keep the spectral convention as it is. We will need to check the input header first and apply proper parameters when exporting.